test/default-level-test.js

```diff
@@ -54,6 +57,7 @@ define(['test/test-helpers'], function(testHelpers) {
                 log.setDefaultLevel("debug");
                 expect(log).toBeAtLevel("debug");
                 expect("debug").not.toBeTheStoredLevel();
+                expect(log.getLevel()).toBe(log.levels.DEBUG);
             });
         });
     });
```

Comment:
These shouldn't just be sprinkled into existing tests. Each test should be testing one specific thing, and each feature (including .getLevel()) should have standalone tests for whatever it's interesting cases are. Here if we accidentally break getLevel then a bunch of random unrelated tests fail, and it's not clear what's gone on and why.

Can you change this to add a couple of new tests for `getLevel()` instead, covering each of the relevant cases for getLevel()? I think those tests should probably be something like:

* Returns WARN initially
* Returns a new set value if setLevel is called
* Returns a new set value if setLevel is called even if it's not persisted

Can't think of any other cases, but feel free to add some if you can :-)